tests/unit/Framework/TestSuiteTest.php

```diff
@@ -227,4 +227,24 @@ public function testCreateTestForConstructorlessTestClass(): void
             ->willReturn(__CLASS__);
         TestSuite::createTest($reflection, 'TestForConstructorlessTestClass');
     }
+
+    /**
+     * @testdox Handles exceptions in tearDownAfterClass()
+     */
+    public function testTearDownAfterClassInTestSuite(): void
+    {
+        $suite = new TestSuite(\ExceptionInTearDownAfterClassTest::class);
+        $suite->run($this->result);
+
+        $this->assertSame(3, $this->result->count());
+        $this->assertCount(1, $this->result->failures());
+
+        /** @var TestFailure $failure */
```

Comment:
Please use `\assert($failure instanceof TestFailure);` instead.

Reply:
As of b880c49bfec871a43702bca8c42b36ccc246f989 neither should be needed anymore.

Reply:
Will that commit also come to 7.5? I have added the `\assert()` for now.